web-ng/src/app/app.module.ts

```diff
@@ -35,6 +35,10 @@ import { ShareDialogModule } from './components/share-dialog/share-dialog.module
 import { HttpClientModule } from '@angular/common/http';
 import { firebase, firebaseui, FirebaseUIModule } from 'firebaseui-angular';
 import { TitleDialogModule } from './components/title-dialog/title-dialog.module';
+import { MatCardModule } from '@angular/material/card';
+import { MatGridListModule } from '@angular/material/grid-list';
+import { CardViewProjectModule } from './components/card-view-project/card-view-project.module';
```

Comment:
"Card view project" doesn't make sense according to English grammar (it implies it's a type of project called a "card view"). Why not just call it "project-list"?

Reply:
updated